### PR TITLE
fix: cusdis iframe theme switch

### PR DESCRIPTION
**Pull request by jukrb0x**
this fixes https://github.com/shuding/nextra/issues/1834 

---

the cusdis theme doesn't change if the blog theme is changed, refresh to make it right.

| Case | Error | Expectation |
| --- | --- | --- |
| light -> dark | image | image|
| dark -> light | image | image |

### CODE REVIEW

File: examples/blog/theme.config.jsx [hunk 1]
```diff
@@ -14,6 +17,9 @@ export default {
       </h1>
     )
   },
+  cusdis: {
+    appId: 'a2d11511-7012-4254-9483-cb49c8f4dfe8'
```
Comment (jukrb0x) [inline examples/blog/theme.config.jsx:21]: Is this `appId` for testing only?

Comment (B2o5T) [reply examples/blog/theme.config.jsx:21]: Yeah, I found in Google just for rendering comments in examples

**Comment by Sam-tech94** [comment]
hey you guys, can u help me, my iframe in my markdown file is not rendered on the browser, can u say why and how to fix it ?

**Comment by B2o5T** [comment]
@Sam-tech94 please do not reply on merged pr, considering create a new issue is better

**Pull request merged**